pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -175,6 +175,17 @@ def test_round(self):
             tm.assertRaisesRegexp(ValueError, msg, rng.round, freq='M')
             tm.assertRaisesRegexp(ValueError, msg, elt.round, freq='M')
 
+            # GH 14440
```

Comment:
yes this is just losing precision - not sure much can be done

we could warn / raise potentially though
would that be useful?